shared/chat/conversation/messages/cards/team-journey/util.tsx

```diff
@@ -37,10 +38,25 @@ function renderWelcomeMessage(
   cannotWrite: boolean
 ): React.ReactNode {
   return (
-    <Kb.Markdown smallStandaloneEmoji={false} lineClamp={3} selectable={false}>
+    <Kb.Markdown
+      smallStandaloneEmoji={false}
+      lineClamp={3}
+      selectable={false}
+      paragraphTextClassName="text_BodySmall"
+      style={welcomeStyle}
```

Comment:
`styleSheetCreate` outputs an obj whose keys are the individual styles. So `const welcomeStyle =` below can be `const styles =` and this can be `styles.welcomeStyle`